SPECS/reaper/reaper.spec

```diff
@@ -185,6 +178,14 @@ fi
 %{_unitdir}/cassandra-%{name}.service
 
 %changelog
+* Thu May 23 2024 Archana Choudhary <archana1@microsoft.com> - 3.1.1-9
+- Repackage and update src UI node modules to 3.1.1-1
+- Address CVE-2024-4068 by upgrading the version of the npm module "braces" to 3.0.3
```

Comment:
If I understand the packaging correctly, the `reaper-srcui-node-modules` tarball contains `braces`. If that's the case, could you also add a line explaining the changes in the `reaper-bower-components` tarball?

Reply:
reaper-bower-components are related to src/ui and therefore were regenerated along with node_modules.